rct229/rulesets/ashrae9012019/section4/section4rule11.py

```diff
@@ -0,0 +1,216 @@
+                    "baseline_hvac_sys_type_ids_dict_b"
+                ]
+                dict_hvac_sys_zones_served_p = data["dict_hvac_sys_zones_served_p"]
+                rmi_b = data["rmi_b"]
+                rmi_p = data["rmi_p"]
+
+                fan_schedule_b = get_aggregated_zone_hvac_fan_operating_schedule(
+                    rmi_b, zone_b["id"], is_leap_year
+                )
+                fan_schedule_p = get_aggregated_zone_hvac_fan_operating_schedule(
+                    rmi_p, zone_p["id"], is_leap_year
+                )
+
+                fan_schedule_hourly_values_b = fan_schedule_b.get("hourly_values")
+                fan_schedule_hourly_values_p = fan_schedule_p.get("hourly_values")
+
+                schedule_match = (
+                    fan_schedule_hourly_values_b == fan_schedule_hourly_values_p
+                )
+
+                proposed_served_by_multizone = False
+                hvac_id_b = zone_b["terminals"][0][
+                    "served_by_heating_ventilating_air_conditioning_system"
+                ]
+                hvac_sys_type_b = baseline_hvac_sys_type_ids_dict_b.keys()[
+                    list(baseline_hvac_sys_type_ids_dict_b.values()).index(hvac_id_b)
+                ]
+
+                baseline_served_by_multizone = hvac_sys_type_b in APPLICABLE_SYS_TYPES
+
+                list_hvac_systems_p = get_list_hvac_systems_associated_with_zone(
+                    rmi_p, zone_p["id"]
+                )
+                for hvac_id_p in list_hvac_systems_p:
+                    if (
+                        len(
+                            list(
+                                dict_hvac_sys_zones_served_p[hvac_id_p][
+                                    "Zone_List"
+                                ].values()
+                            )
+                        )
+                        > 1
+                    ):
+                        for terminal_p in zone_p["terminals"]:
+                            if (
+                                terminal_p[
+                                    "served_by_heating_ventilating_air_conditioning_system"
+                                ]
+                                == hvac_id_p
+                            ):
+                                if terminal_p["heating_capacity"] > 0:
+                                    proposed_served_by_multizone = True
+
+                system_type_match_baseline_proposed = (
+                    proposed_served_by_multizone == baseline_served_by_multizone
+                )
+
+                floor_name_b = zone_b["floor_name"]
+                dict_of_zones_hvac_systems_serving_specific_floor_b = (
+                    get_dict_of_zones_hvac_sys_serving_specific_floor(
+                        floor_name_b, rmi_b
+                    )
+                )
+
+                list_hvac_sys_serving_floor_b = list(
+                    dict_of_zones_hvac_systems_serving_specific_floor_b.values()
```

Comment:
The return type is `{zone_id: [hvac_id_list]}`. The above code will result in a 2 dimension list. for example:
```python
list_hvac_sys_serving_floor_b = [["sys_1", "sys_2"], ["sys_1", "sys_3"]]
```
You need to flatten this list - turn to 2 dimension into a list first, then do the loop below. - examples can be found using chain function from the itertools package.

Please verify the implementation in the actual function instead of just copying RDS. This mistake has repeatedly found in this PR.